tests/components/incomfort/test_init.py

```diff
@@ -0,0 +1,102 @@
+"""Tests for Intergas InComfort integration."""
+
+from datetime import timedelta
+from unittest.mock import MagicMock, patch
+
+from aiohttp import ClientResponseError
+from incomfortclient import IncomfortError
+import pytest
+from syrupy import SnapshotAssertion
+
+from homeassistant.components.incomfort import DOMAIN
+from homeassistant.components.incomfort.coordinator import UPDATE_INTERVAL
+from homeassistant.const import STATE_UNAVAILABLE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+from homeassistant.util.dt import utcnow
+
+from .conftest import MOCK_CONFIG
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+
+async def test_setup_platforms(
```

Comment:
Honestly, we could put these tests in a preliminary to make this one smaller

Reply:
The tests are coordinator specific. But I can have a look.

Reply:
Done in #118827